### DISCUSSION TITLE
Incorporate Web3Auth

### DISCUSSION THREAD

**Discussion by dev-cz** [wevm/wagmi#811] [Ideas]
Would be nice to see Web3Auth working with wagmi for Social media login.

https://web3auth.io/

**Comment by 3scava1i3r** [comment]
Agreed working on it for some time now :)

**Comment by tmm** [comment]
The [Magic Connector](https://github.com/EveripediaNetwork/wagmi-magic-connector) seems similar if you want to use it as an example.

**Reply by 3scava1i3r** [reply]
Thanks, will check that out
Do i need to make a different package for the same or just do a pr?

**Reply by dev-cz** [reply]
> The [Magic Connector](https://github.com/EveripediaNetwork/wagmi-magic-connector) seems similar if you want to use it as an example.

Hmm. Will check it out. Any examples of it working with wagmi?

**Reply by tmm** [reply]
Some projects that are using it

https://github.com/EveripediaNetwork/wagmi-magic-connector/network/dependents?package_id=UGFja2FnZS0zMTA2ODU3NTU4

**Reply by 3scava1i3r** [reply]
> Some projects that are using it
> 
> https://github.com/EveripediaNetwork/wagmi-magic-connector/network/dependents?package_id=UGFja2FnZS0zMTA2ODU3NTU4

surely while checking more on the package i see good resemblance sorting some types,other that majorly done should i post a pr then or make a different package like them?

**Reply by dev-cz** [reply]
Looks pretty legit. Got it tested out but running into a few issues with wagmi (no rainbowkit) after authorizing with Twitter. Doesnt redirect and says user reject request on wallet selection. Hmm.

**Comment by recidive** [comment]
It's here https://github.com/Web3Auth/web3auth-wagmi-connector !!
